### PR TITLE
Transfer: forbid long Payment IDs

### PR DESCRIPTION
**Pull request by xiphon**

### CODE REVIEW

File: pages/Transfer.qml [hunk 1]
```diff
@@ -394,8 +394,10 @@ Rectangle {
       MoneroComponents.WarningBox {
           // @TODO: remove after pid removal hardfork
           id: paymentIdWarningBox
-          text: qsTr("You can enable transfers with payment ID on the settings page.") + translationManager.emptyString;
-          visible: !persistentSettings.showPid && (warningLongPidTransfer || warningLongPidDescription)
+          text: qsTr("Long payment IDs are obsolete. \
+          Long payment IDs were not encrypted on the blockchain and would harm your privacy. \
```
Comment (xmrdsc) [inline pages/Transfer.qml]: *could*

Comment (xiphon) [reply pages/Transfer.qml]: The message comes from https://github.com/monero-project/monero/pull/5834

File: pages/Transfer.qml [hunk 2]
```diff
@@ -360,7 +359,7 @@ Rectangle {
           }
 
           ColumnLayout {
-              visible: appWindow.persistentSettings.showPid || paymentIdCheckbox.checked
+              visible: paymentIdCheckbox.checked
```
Comment (selsta) [inline pages/Transfer.qml]: How is this supposed to be visible?

Comment (xiphon) [reply pages/Transfer.qml]: To support Monero Payment URI decoding

Comment (selsta) [reply pages/Transfer.qml]: Ah makes sense

**Pull request merged**